packages/cli/src/util/command.ts

```diff
@@ -1,20 +1,10 @@
 import { Command } from 'commander';
 
-class MedplumCommand extends Command {
-  createCommand(name: string | undefined): Command {
```

Comment:
This createCommand works only with `.command`. `rest` and `auth` commands aren't created this way.